pkg/image/builder.go

```diff
@@ -496,11 +493,7 @@ func IsPrivateImage(image string, dockerHubRegistry registrytypes.RegistryOption
 			continue
 		}
 
-		if isTooManyRequests(err) {
-			lastErr = err
-			isRateLimited = true
-			continue
-		}
+		logger.Infof("Marking image '%s' as private because: %v", image, err.Error())
```

Comment:
this will help make debugging replicated proxy and docker rate limit issues easier